sormas-e2e-tests/src/test/resources/features/sanity/web/Contacts.feature

```diff
@@ -247,3 +247,31 @@ Feature: Contacts end to end tests
     And I select Travel option in Type of activity from Combobox in Exposure form
     Then I fill Location form for Type of place by Facility option
     And I click on save button from Epidemiological Data
+
+  @issue=SORDEV-5670
+  Scenario: Fill the follow-up tab
```

Comment:
please change the name of the scenario to reflect the purpose of the test

Reply:
This test is not written by myself, so I can't change the name of this scenario